mappings/net/minecraft/entity/player/PlayerModelParts.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS none/aal net/minecraft/entity/player/PlayerModelParts
```

Comment:
Client-side-only classes belong in client/. In this case, probably client/model/, which I had just created.